src/commands/variables/create.ts

```diff
@@ -38,7 +38,19 @@ export default class CreateVariable extends CreateCommand {
         }
         flags._feature = feature
 
-        const params = await this.populateParametersWithZod(CreateVariableDto, this.prompts, flags)
+        let params = await this.populateParametersWithZod(CreateVariableDto, this.prompts, flags)
+        if (!headless) {
+            const { attachToFeature } = await inquirer.prompt([{
+                name: 'attachToFeature',
+                message: 'Attach to a feature?',
```

Comment:
Do you think it would make sense for `attach to` to instead be `associate with`? Only because we use the term "associated" in the dashboard/docs, otherwise I wouldnt pick this nit

Reply:
yes I'll switch to associate